src/components/monitor/monitor.jsx

```diff
@@ -13,43 +19,74 @@ const categories = {
     list: '#FC662C'
 };
 
+const types = {
+    default: DefaultMonitor,
+    large: LargeMonitor
+};
+
 const MonitorComponent = props => (
-    <Draggable
-        bounds="parent"
-        defaultClassNameDragging={styles.dragging}
-        onStop={props.onDragEnd}
-    >
-        <Box
-            className={styles.monitor}
-            componentRef={props.componentRef}
+    <ContextMenuTrigger id={`monitor-${props.label}`}>
+        <Draggable
+            bounds=".monitor-overlay" // Class for monitor container
+            defaultClassNameDragging={styles.dragging}
+            onStop={props.onDragEnd}
         >
-            <Box className={styles.label}>
-                {props.label}
-            </Box>
             <Box
-                className={styles.value}
-                style={{background: categories[props.category]}}
+                className={styles.monitorContainer}
+                componentRef={props.componentRef}
+                onDoubleClick={props.onNextType}
```

Comment:
I noticed some odd behavior in chrome where double click isn't recognized if you double click twice in a row without moving the mouse in-between. But I don't think this PR needs to fix that

Reply:
Yeah it seems like there is also some odd double-click behavior on touch. I think it'll have to be fine-tuned a bit.